wooz1ewu/ITechArt_Front-End
# Nav menu, about, shared-notes and not found pages implemented

# Overview  

> The goal of pr was to implement pages' navigation.  

# Changes  

> 1. PageLayout common component was added to provide common layout(with nav menu) for all the pages.  
> 1. NavMenu component added.  
> 1. Pages routing implemented.  
> 1. Redirecting to not-found page implemented.
> 1. Shared-notes page added with an empty NoteList.  
> 1. About page added.  
> 1. My-notes page set as default.  

# How to test  

> 1. Visit [page](https://wooz1ewu.github.io/ITechArt_Front-End/).
> 1. Try to open menu by pressing sandwich-button.  
> 1. Try to navigate all the provided pages. by clicking on certain button-links.  
> 1. Check if pages' content is available.  
> 1. Try to navigate fake url to check if app redirects you to not-found page.
